docs/doc/90-contributing/08-how-to-write-scalar-functions.md

```diff
@@ -23,16 +23,12 @@ Scalar functions (sometimes referred to as User-Defined Functions / UDFs) return
 └─────┘                    └──────┘
 ```
 
-
 ### Knowledge before writing the eval function
 
 #### Logical datatypes and physical datatypes.
 
 Logical datatypes are the datatypes that we use in Databend, and physical datatypes are the datatypes that we use in the execution/compute engine.
-Such as `Date32`, it's a logical data type, but its physical is `Int32`, so its column is represented by `Int32Column`.
-
-We can get logical datatype by `data_type` function of `DataField` , and the physical datatype by `data_type` function in `ColumnRef`.
-`ColumnsWithField` has `data_type` function which returns the logical datatype.
+Such as `Date`, it's a logical data type, but its physical is `Int32`, so its column is represented by `Buffer<i32>`.
```

Comment:
Take `Date` as an example, `Date` is a logical datatype while its physical datatype is `Int32`, so ........